### PR TITLE
flowchart for writeMotor program added with code (issue #37)

### LINKED ISSUE #37: Add flowchart for writeMotor program

### LINKED ISSUE #45: Add license to the flowchart codes.

### PR DESCRIPTION
**Pull request by addy1997**

### CODE REVIEW

File: utils/writeMotor flowchart/code/code.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02111, USA.
+ */
+
+writeMotor;
+Initialize the variables to store Left/Right Motor's analog output;
+branch(left_motor)
```
Comment (sarthakahuja11) [inline utils/writeMotor flowchart/code/code.txt:23]: Does this write motor flow get involved with the compute angle flow in anyway? 
Just wanted to understand if the analog output has any relation with RSSI converted distance/compute angle stuff or it is completely independent or dependent on something else?

I think the main flow diagram you made might depict this..

**Comment by addy1997** [comment]
Nope, it doesn't get involved.

First we compute the angle and then we use the writeMotor code

**Comment by sarthakahuja11** [comment]
Yes it's in the main flowchart I think. Need another coherent one i think once we add Bluetooth and other stuff

**Comment by addy1997** [comment]
Sure, will make the changes after the final version if you don't mind.

**Pull request merged**